portfolio/src/main/webapp/script.js

```diff
@@ -106,22 +106,59 @@ function createComment(element) {
   liComment.innerText = element.key + ": " + element.value;
   liComment.className = "commentli";
   liComment.id = element.key + element.value;
-  const deleteComment = document.createElement('button');
-  deleteComment.innerText = "X";
-  deleteComment.className = "singleDelete";
+  const deleteComment = addDeleteButton();
   liComment.appendChild(deleteComment);
   deleteComment.addEventListener("click", function() {
     deleteSingleComment(liComment.id);
   });
+  const reactToComment = addReactionButton();
+  liComment.appendChild(reactToComment);
+  const reactionContainer = document.createElement('div');
+  reactionContainer.className = "reactionContainer";
+  liComment.appendChild(reactionContainer);
+  reactToComment.addEventListener("change", function() {
+    const reaction = addReaction(reactToComment.options[reactToComment.selectedIndex].innerHTML);
+    reactionContainer.appendChild(reaction);
+  });
   return liComment;
 }
 
+function addReaction(emoji) {
+  const reaction = document.createElement('p');
+  reaction.innerHTML = emoji;
+  reaction.className = "reaction";
+  return reaction;
+}
+
+function addDeleteButton(){
+  const deleteComment = document.createElement('button');
+  deleteComment.innerText = "X";
+  deleteComment.className = "singleDelete";
+  return deleteComment;
+}
+
+function addReactionButton(){
+  const addReaction = document.createElement('select');
+  addReaction.className = "reactionButton";
+  addReaction.id = "reactionButton";
+  const cover = document.createElement('option');
+  cover.innerText = "!!";
+  cover.disabled = true;
+  cover.selected = true;
+  const happy = document.createElement('option');
+  happy.innerHTML = String.fromCodePoint(0x1F601);
+  const sad = document.createElement('option');
+  sad.innerHTML = String.fromCodePoint(0x1F622);
+  addReaction.appendChild(cover);
+  addReaction.appendChild(happy);
+  addReaction.appendChild(sad);
+  return addReaction;
+}
+
```

Comment:
The function names should reflect what they do. In this case, these all create and return new DOM elements rather than add them anywhere in particular.